src/strand.rs

```diff
@@ -0,0 +1,62 @@
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum Strand {
+    Positive,
+    Negative
+}
+impl Strand {
+    fn value(&self) -> char {
```

Comment:
For functions like this the best way is to implement `From`:
```rust
impl From<Strand> for char {
    fn from(value: Strand) -> char {
        // the same function body here
    }
}
```
But in which cases you will need to directly use the char?
If it's just for printing you can implement `Display`:
```rust
impl std::fmt::Display for Structure {
    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
       let character = // convert self to a char or str;
       write!(f, "{}", character)
    }
}
```
so you can do stuff like:
```rust
println!("{}", Strand::Positive);
```